### PR TITLE
[PURCHASE-1236] Add edition info to LAI interface

### PR DESCRIPTION
**Pull request by ashfurrow**
This adds an artwork's edition info to the LAI interface under "Lot Info":

Screen Shot 2019-07-01 at 3 37 15 PM

This PR is a work-in-progress because we're waiting for a finalized design.

Lily and I also changed how the dimensions are displayed: users with their locale set to use the metric system will now see `cm` values, and others see `in` values. This is a change from before, when _everyone_ saw `cm` values 😛 /cc @alloy

### CODE REVIEW

**Comment by alloy** [comment]
I’m in favour of that change, regardless of how superior the metric system is 😛

**Comment by ashfurrow** [comment]
Here's what the finalized design looks like:

Screen Shot 2019-07-02 at 3 22 11 PM

File: Artsy/View_Controllers/Live_Auctions/ViewModels/LiveAuctionLotViewModel.swift [hunk 1]
```diff
@@ -151,13 +155,21 @@ class LiveAuctionLotViewModel: NSObject, LiveAuctionLotViewModelType {
     }
 
     var lotArtworkDimensions: String? {
-        return model.artwork.dimensionsCM
+        if self.usersLocale.usesMetricSystem {
+            return model.artwork.dimensionsCM
+        } else {
+            return model.artwork.dimensionsInches
+        }
```
Comment (alloy) [inline Artsy/View_Controllers/Live_Auctions/ViewModels/LiveAuctionLotViewModel.swift]: Not something we should do now, but ideally we wouldn’t even fetch unneeded details–i.e. use the locale defaults at query time instead.

File: Artsy_Tests/Networking_Tests/LiveAuctionLotViewModelSpecs.swift [hunk 2]
```diff
@@ -94,5 +94,31 @@ class LiveAuctionLotViewModelSpec: QuickSpec {
...
+                subject = LiveAuctionLotViewModel(lot: lot!, bidderCredentials: creds)
+            }
+
+            it("works for inches") {
+                subject.usersLocale = Locale(identifier: "en-US")
+                expect(subject.lotArtworkDimensions) == "2x2in"
+            }
+
+            it("works for centimeters") {
+                subject.usersLocale = Locale(identifier: "en-CA")
```
Comment (alloy) [inline Artsy_Tests/Networking_Tests/LiveAuctionLotViewModelSpecs.swift]: 🇨🇦 🤘

File: Artsy/View_Controllers/Live_Auctions/ViewModels/LiveAuctionLotViewModel.swift (hunk 1 above)
Comment (ashfurrow) [reply Artsy/View_Controllers/Live_Auctions/ViewModels/LiveAuctionLotViewModel.swift]: Oh yeah, that'd be cool!

**Pull request merged**